### ISSUE TITLE
Problem with restoring display mode setting after gnome-shell restart

### ISSUE THREAD

**Issue by SuperMeowBro** [andyholmes/gnome-shell-extension-gsconnect#803]
**Describe the bug**

There is a problem with restoring display mode setting after gnome-shell restart. If before the restart display mode was set to menu, both are active after gnome-shell restart as you can see in screenshot.

**Steps To Reproduce:**

1. Go to settings
2. Set display mode to 'menu' not 'panel'
4. Press Alt+F2, type 'r' and press enter

**Expected behavior**

Setting for display mode should be restored correctly after gnome-shell restart. Only the set display mode should be active then.

**Screenshots**

![grafik](https://user-images.githubusercontent.com/21910440/77970327-2cf01000-72ec-11ea-9c7b-8fb0951fbccc.png)

**Support Log**

-/-

**System Details (please complete the following information):**

 - **GSConnect version:** 36
   - **Installed from:** GNOME Extensions Website
 - **GNOME/Shell version:** 3.36
 - **Distro/Release:** Manjaro

**GSConnect environment (if applicable):**

-/-

**Additional Notes:**

-/-

**Comment by andyholmes** [comment]
I can't reproduce this. Are you using another extension that modifies the panel or indicators?

**Comment by SuperMeowBro** [comment]
You are right. I'have checked it again by disabling all extensions one by one and it seems that the problem is caused by "Dash to Panel"-Extension. With gnome-shell panel there is no issue.

**Comment by ferdnyc** [comment]
That would make this a duplicate of #725, but the good news is that the dash-to-panel developers addressed the issue (they believe, though the original reporter never confirmed it) after it was reported in home-sweet-gnome/dash-to-panel#829. So, it may just be a matter of updating the other extension.

**Comment by ferdnyc** [comment]
(I should point out that I don't know whether that fix was actually _released_ in dash-to-panel yet, so it may require updating to a version that isn't yet available on extensions.gnome.org. The dash-to-panel developers would be the best ones to consult on that.)

**Comment by andyholmes** [comment]
Ah right, I totally forgot this came up before. I think they've had a few releases since, but maybe they were waiting on confirmation before they merged.
